doc/reference/src/main/java/org/crsh/doc/Generator.java

```diff
@@ -73,7 +73,7 @@ public static void main(String[] args) throws Exception {
     }
     if (buffer.length() > 0) {
       File f = new File(root, "reference.asciidoc");
-      PrintWriter pw = new PrintWriter(f);
+      PrintWriter pw = new PrintWriter(f, "UTF-8");
```

Comment:
It would be better if this was settable, rather than forcing an encoding, especially if the default charset is not desired.